docker-compose.yml

```diff
@@ -171,14 +171,14 @@ services:
       - data_rabbitmq:/var/lib/rabbitmq
 
   autograph:
-    image: mozilla/autograph:7.3.3
+    image: mozilla/autograph:7.3.6@sha256:abbd38f0969b86fca2266570870087c15db91204533b893cc2a948c422ed3424
     command: /go/bin/autograph -c /data/autograph/autograph_localdev_config.yaml
     volumes:
       - data_autograph:/data/autograph
 
   addons-frontend:
     <<: *env
-    image: mozilla/addons-frontend:latest
+    image: mozilla/addons-frontend:2025.03.20-1@sha256:32b1c20b6678df73df4ac81c19cbb74bfe5a062a7e39273efba626d0347e8019
```

Comment:
Figured our last tag wasn't a bad target for addons-frontend rather than `latest`. We typically don't need the latest and greatest addons-frontend when working with addons-server, just _some_ working frontend.